scripts/performance/perf_spike_load.py

```diff
@@ -1,79 +1,206 @@
 #! /usr/bin/env python3
 
-"""This script uses another load script (perf_processes.py) running it with different parameters which are
-provided in config_perf_spike_load.yml file"""
 
-
-from datetime import timedelta, datetime
 import subprocess
 import yaml
 import time
 import os
+import collections
+import matplotlib.pyplot as plt
+import numpy as np
+import argparse
+
+parser = argparse.ArgumentParser(description='This script uses another load script (perf_processes.py) running it '
+                                             'with different parameters which are provided in '
+                                             'config_perf_spike_load.yml file')
+
+parser.add_argument('-g', '--graph', default=False, type=bool, required=False, dest='graph',
+                    help='Build a graph to check if the configured profile is correct')
+
+args = parser.parse_args()
+
+
+def create_output_directory(folder_path):
+    output_folder = ""
+    for folder in folder_path:
+        output_folder = os.path.join(output_folder, folder)
```

Comment:
I think it is not need to use path.join here since func arg is the folder from config. just expand user is enough.